pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -630,6 +680,19 @@ private WebSocketConfiguration webSocketConfiguration() {
   }
 
   MetricsConfiguration metricsConfiguration() {
+
+    CommandLineUtils.checkOptionDependencies(
```

Comment:
This changed in #678 - here's the new rules

```suggestion
    CommandLineUtils.checkOptionDependencies(
        logger,
        commandLine,
        "--metrics-enabled",
        !isMetricsEnabled,
        Arrays.asList(
            "--metrics-mode",
            "--metrics-host",
            "--metrics-port"));
            
    CommandLineUtils.checkOptionDependencies(
        logger,
        commandLine,
        "--metrics-push-enabled",
        !isMetricsEnabled,
        Arrays.asList(
            "--metrics-push-host",
            "--metrics-push-port",
            "--metrics-push-interval",
            "--metrics-push-prometheus-job"));
```